packages/docs/src/pages/sx-prop.mdx

```diff
@@ -344,15 +360,13 @@ which support the `sx` prop, in an MDX file as a workaround.
 ```js
 import { Box } from 'theme-ui'
 
-<Box
+;<Box
   sx={{
     padding: 3,
     bg: 'highlight',
   }}
 >
-
-# Hello
-
+  # Hello
```

Comment:
Wait, this doesn't work, does it?

If you don't wrap markdown in empty lines you're still in JSX, so this won't render a heading, and that # is just printed. I mean, I know that's just a code snippet in the docs, but still.

Reply:
You're correct, I believe. I guess we'll just counter Prettier here & figure out how to avoid it in the future.